src/Microsoft.ML.Core/Data/VBuffer.cs

```diff
@@ -424,11 +424,6 @@ public static void Copy(T[] src, int srcIndex, ref VBuffer<T> dst, int length)
             dst = new VBuffer<T>(length, values, dst.Indices);
         }
 
-        public static void Copy(ref VBuffer<T> src, ref VBuffer<T> dst)
```

Comment:
Note: there was only 1 usage of this method, and it is just a wrapper over `src.CopyTo(ref dst)`, so I thought it was best to drop it in favor of only having a single way to copy from one buffer to another.

Let me know if this was a bad idea.

Reply:
It's also used once in internal repos, so I think this is a good change.

---
In reply to: [229444488](https://github.com/dotnet/machinelearning/pull/1454#discussion_r229444488) [](ancestors = 229444488)